README.md

```diff
@@ -21,7 +21,7 @@ npm i puppeteer-page-proxy
 - `proxy` <[string](https://developer.mozilla.org/en-US/docs/Glossary/String)|[object](https://developer.mozilla.org/en-US/docs/Glossary/Object)> Proxy to use in the current page.
   * Begins with a protocol (e.g. http://, https://, socks://)
   * In the case of [proxy per request](https://github.com/Cuadrix/puppeteer-page-proxy#proxy-per-request), this can be an object with optional properties for overriding requests:\
-`url`, `method`, `postData`, `headers`\
+`url`, `method`, `postData`, `headers`, `ignoreInvalidCookies`\
```

Comment:
Is `ignoreInvalidCookies` really necessary?

Reply:
I wouldn't mind setting it to true by default and not having it as an option, but I thought you may like it this way. I'll change it then.